EIPS/eip-2935.md

```diff
@@ -66,9 +59,11 @@ def resolve_blockhash(block: Block, state: State, arg: uint64):
   return state.load_slot(HISTORY_STORAGE_ADDRESS, arg % HISTORY_SERVE_WINDOW)
 ```
 
+<!-- TODO: Do we need to update the implementation because there is not history backfilling? -->
+
 ### Contract Implementation
 
-Exact evm assembly that can be used for the contract to resolve `BLOCKHASH`
+Exact evm assembly that can be used for the contract to resolve `BLOCKHASH` from verkle_block_fork
```

Comment:
that's not correct: the contract is available before the prague fork, whether it resolves blockhash or not.

Reply:
so I would just remove the reference to `BLOCKHASH`